### PR TITLE
Clean up todos and stale comments

### PR DESCRIPTION
**Pull request by Bridgerz**

### CODE REVIEW

File: contracts/BridgeVault.sol [hunk 1]
```diff
@@ -49,7 +49,6 @@ contract BridgeVault is Ownable, IBridgeVault {
         // Unwrap the WETH
         wETH.withdraw(amount);
 
-        // TODO: check transfer success
```
Comment (longbowlu) [inline contracts/BridgeVault.sol:52]: don't we need to check transfer success or the following line will revert?

File: contracts/utils/BridgeMessage.sol [hunk 2]
```diff
@@ -163,12 +142,7 @@ library BridgeMessage {
 
         // extract target address from payload (35-54)
         address targetAddress;
-        // why `add(targetAddressLength, offset)`?
-        // At this point, offset = 35, targetAddressLength = 20. `mload(add(payload, 55))`
-        // reads the next 32 bytes from bytes 23 in paylod, because the first 32 bytes
-        // of payload stores its length. So in reality, bytes 23 - 54 is loaded. During
-        // casting to address (20 bytes), the least sigificiant bytes are retained, namely
-        // `targetAddress` is bytes 35-54
```
Comment (longbowlu) [inline contracts/utils/BridgeMessage.sol]: can we keep these comments? also line 120-127 and line 185-190. I hope it's useful for future readers

File: contracts/BridgeVault.sol (hunk 1 above)
Comment (Bridgerz) [reply contracts/BridgeVault.sol:52]: both `address.tranfser` and `wETH.withdraw` either succeeds or reverts so its not necessary

File: contracts/utils/BridgeMessage.sol (hunk 2 above)
Comment (Bridgerz) [reply contracts/utils/BridgeMessage.sol]: sure! I'll add them back in.

**Review by longbowlu — APPROVED** [review]

**Pull request merged**